membership-attribute-service/app/services/AttributesFromZuora.scala

```diff
@@ -73,11 +74,12 @@ class AttributesFromZuora(implicit val executionContext: ExecutionContext) exten
     //return what we know from Dynamo if Zuora returns an error
     val fallbackIfZuoraFails: Future[(String, Option[Attributes])] = attributesFromDynamo map { maybeDynamoAttributes => ("Dynamo", maybeDynamoAttributes map DynamoAttributes.asAttributes)}
 
+    val what: Future[Disjunction[String, Future[Option[ZuoraAttributes]]]] = attributesFromZuora
     val attributesFromZuoraUnlessFallback: Future[(String, Option[Attributes])] = attributesFromZuora flatMap {
       case -\/(error) => fallbackIfZuoraFails
       case \/-(maybeZuoraAttributes) => maybeZuoraAttributes flatMap { zuoraAttributes: Option[ZuoraAttributes] =>
         attributesFromDynamo map { dynamoAttributes =>
-          val combinedAttributes = AttributesMaker.zuoraAttributesWithAddedDynamoFields(zuoraAttributes, dynamoAttributes)
+          val combinedAttributes = zuoraAttributes.map(ZuoraAttributes.asAttributes(_))
```

Comment:
I don't think combined attributes is an appropriate name for this anymore?

Also, if I've read it correctly I think we're always setting membership number to None here... is that intentional?

Reply:
Good points. 

I'm going to put the code back in to maintain membership number for now. It would make sense to deal with membership number in its own PR.